src/App.tsx

```diff
@@ -1,34 +1,39 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { TodoContextProvider } from './components/TodoContext';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [isModelOpen, setIsModelOpen] = useState(false);
+
+  const [isLoadingTodos, setIsLoadingTodos] = useState(true);
+
   return (
-    <>
+    <TodoContextProvider turnOffLoad={setIsLoadingTodos}>
```

Comment:
it's good that you know how to use Context, but you don't need it in this project. Please read this [article](https://blog.logrocket.com/pitfalls-of-overusing-react-context/)